app/presenters/risc_configuration_presenter.rb

```diff
@@ -0,0 +1,22 @@
+# https://openid.net/specs/openid-risc-profile-1_0-ID1.html#discovery
+class RiscConfigurationPresenter
+  include Rails.application.routes.url_helpers
+
+  DELIVERY_METHOD_PUSH = 'https://schemas.openid.net/secevent/risc/delivery-method/push'.freeze
+
+  def configuration
+    {
+      issuer: root_url,
+      jwks_uri: api_openid_connect_certs_url,
+      delivery_methods_supported: [
+        DELIVERY_METHOD_PUSH,
+      ],
+      delivery: [
+        {
+          delivery_method: DELIVERY_METHOD_PUSH,
+          url: api_security_events_url,
+        },
+      ]
```

Comment:
Adding a `delivery` clause here was something I kind of made up...

The spec allows for dynamic registration of various endpoints, and the responses there contain sections like this: https://openid.net/specs/openid-risc-profile-1_0-ID1.html#rfc.section.4.1.2.1

The [section on HTTP Push](https://openid.net/specs/openid-risc-profile-1_0-ID1.html#rfc.section.5.2.1.1) (which we support) describes these parts, so I just figured they seemed like a good thing to add because the [HTTP Push Doc](https://tools.ietf.org/html/draft-ietf-secevent-http-push-00) doesn't seem to include anything about discovery, just how the endpoint should act